Base.rte/Craft/StorageCrate/DropCrate.lua

```diff
@@ -1,23 +1,41 @@
 function Create(self)
-	self.GibTimer = Timer()
-	
+	self.GibTimer = Timer();
+	-- Choose horizontal direction
+	local dir = 1;
 	if math.random() > 0.5 then
-		self.AngularVel = -math.random(6, 10)
-		self.Vel = Vector(math.random(2, 5), 20)
-	else
-		self.AngularVel = math.random(6, 10)
-		self.Vel = Vector(-math.random(2, 5), 20)
+		dir = -1;
 	end
+	-- Randomize velocities
+	self.AngularVel = -dir * math.random(1, 5);
+	self.Vel = Vector(dir * math.random(5, 20), 30);
 end
-
 function Update(self)
-	if self.GibTimer:IsPastSimMS(1000) then
-		self:GibThis()
-	elseif self.Vel.Largest > 5 then
-		self.GibTimer:Reset()
+	-- Apply damage to the actors inside based on impulse forces
+	if self.TravelImpulse.Magnitude > self.Mass then
```

Comment:
I think it was just an elaborate reference ("protective strength" of the crate?) to activate the damage script. I guess it could be a static number?

Reply:
Ahh okay. I don't mind it being related to mass (or not being related to mass for that matter), I just wasn't sure why mass worked directly with travel impulse magnitude.
Probably the best way to do it would be make it related to gold cost, so if there are more expensive dropcrates they can handle heavier loads without damaging their contents.